crowbar_framework/lib/remote_node.rb

```diff
@@ -111,6 +111,12 @@ def chef_clients_running?(host_or_ip)
   # Workaround for similar issue
   # http://projects.puppetlabs.com/issues/2776
   def resolve_host(host)
+    sleep_time = 5
+    timeout_at = Time.now + 60
+    msg = "Waiting to resolve hostname '#{host}'; next attempt in #{sleep_time} seconds until #{timeout_at}"
+    wait_until(msg, timeout_at.to_i, sleep_time, {}) do
+      !`dig +short #{host} | head -n1`.rstrip.empty?
+    end
```

Comment:
You could store `dig +short #{host} | head -n1`.rstrip in a variable here and avoid the extra dig call after the loop

Reply:
> Please trim the boilerplate text if you are not using it!

@a

> Please trim the boilerplate text if you are not using it!

@aspiers done. But it's the commit msg that is important! :)

Reply:
Done